### PR TITLE
Update config_impl.cc

### PR DESCRIPTION
**Pull request by luisotero**
Removing brackets from default-initialization of protocol buffer in HedgePolicyImpl.
Signed-off-by: Luis Otero <lusimi86@gmail.com>

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Description: Removing unnecessary brackets during default-initialization of a protocol buffer in HedgePolicyImpl. These brackets would cause build failures if the prototbuf in question ever manages to be arena-enabled.
Risk Level: Low
Testing:
Docs Changes:
Release Notes:
[Optional Fixes #Issue]
[Optional Deprecated:]

### CODE REVIEW

File: source/common/router/config_impl.cc [hunk 1]
```diff
@@ -62,7 +62,7 @@ HedgePolicyImpl::HedgePolicyImpl(const envoy::api::v2::route::HedgePolicy& hedge
       hedge_on_per_try_timeout_(hedge_policy.hedge_on_per_try_timeout()) {}
 
 HedgePolicyImpl::HedgePolicyImpl()
-    : initial_requests_(1), additional_request_chance_({}), hedge_on_per_try_timeout_(false) {}
+    : initial_requests_(1), additional_request_chance_(), hedge_on_per_try_timeout_(false) {}
```
Comment (mattklein123) [inline source/common/router/config_impl.cc]: Can you just remove this initializer entirely and rely on the default?

Comment (luisotero) [reply source/common/router/config_impl.cc]: Thanks!

**Comment by mattklein123** [comment]
@luisotero I think the merge didn't go well, there is no diff now. :(

/wait

**Comment by luisotero** [comment]
> @luisotero I think the merge didn't go well, there is no diff now. :(
> 
> /wait

Sorry about that, Matt! Let me know if this looks better.
Thanks!,
-luis

**Pull request merged**